astroid/nodes/node_classes.py

```diff
@@ -793,11 +788,7 @@ def arguments(self):
         return list(itertools.chain((self.posonlyargs or ()), self.args or ()))
 
     def format_args(self, *, skippable_names: set[str] | None = None) -> str:
-        """Get the arguments formatted as string.
-
-        :returns: The formatted arguments.
```

Comment:
Please keep the docstring. Again, this is only one on many instances in this PR. Please take a look at it again. _It's fine to remove just the type from the docstring if it doesn't add any more value._